src/lib/vm/vm.js

```diff
@@ -1963,6 +1964,10 @@ export default class VM {
         </div>
       );
     }
+    if (!this.alive) {
+      return <div className="alert alert-danger">VM is dead</div>;
+    }
```

Comment:
Not sure if this is needed, but it's better to return it than run an empty code.